lib/apiservers/engine/backends/container.go

```diff
@@ -349,30 +349,27 @@ func (c *Container) ContainerRename(oldName, newName string) error {
 func (c *Container) ContainerResize(name string, height, width int) error {
 	defer trace.End(trace.Begin(name))
 
-	// save the name provided to us so we can refer to it if we have to return an error
-	paramName := name
```

Comment:
That's the case everywhere, the value in `name` was just saved off so the `name` identifier could be reused -- don't reuse identifiers! It's really confusing!